src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -631,6 +633,16 @@ public void clearAttributes() {
 
     @Override
     public GATKRead copy() {
+        // workaround until https://github.com/google/google-http-java-client/issues/293 is released
+        genomicsRead.setAlignedQuality(new ArrayList<>(genomicsRead.getAlignedQuality()));
+        Map<String, List<String>> infoCopy = new HashMap<>();
+        for (Map.Entry<String, List<String>> entry : genomicsRead.getInfo().entrySet()) {
+            infoCopy.put(entry.getKey(), new ArrayList<>(entry.getValue()));
+        }
+        genomicsRead.setInfo(infoCopy);
+        genomicsRead.getAlignment().setCigar(new ArrayList<>(genomicsRead.getAlignment().getCigar()));
+        // end workaround
```

Comment:
This workaround shouldn't be necessary given https://github.com/broadinstitute/hellbender/pull/801. Could you please delete it and confirm that tests still pass?